### PR TITLE
feat: Add create view. And do some project clean up

### PR DESCRIPTION
**Pull request by hindo**
Make adjustments to Republic view. Use Republic type background in republic view

### CODE REVIEW

File: components/Republic/Background.tsx [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
 }
 
-export const Background = ({ url }: { url: string }) => {
-  return <div>{renderBackground({ url })}</div>
+const ColorOverlay = tw.div`absolute inset-0 bg-vlr-dark/40 z-10`
+
+export const Background = ({ republicType, useOverlay = false }: { republicType: Republics; useOverlay?: boolean }) => {
+  return (
+    <div>
+      {useOverlay && <ColorOverlay />}
```
Comment (adrianczekaj) [inline components/Republic/Background.tsx:23]: I thought about adding gradient permanently, but this is brilliant :)

File: components/Republic/Background.tsx [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
+      <Image
+        src={republicBackground[republicType]}
```
Comment (adrianczekaj) [inline components/Republic/Background.tsx:25]: it's nice too
I don't know how to go outside the thinking only about 'switch'

File: components/Republic/Background.tsx [hunk 3]
```diff
@@ -1,20 +1,33 @@
...
+import BlockchainBackground from '../../assets/images/blockchain-background.webp'
+import FrontendBackground from '../../assets/images/frontend-background.webp'
+import MlBackground from '../../assets/images/ml-background.webp'
+import SeoBackground from '../../assets/images/seo-background.webp'
+
+const republicBackground = {
+  [Republics.blockchain]: BlockchainBackground,
+  [Republics.seo]: SeoBackground,
+  [Republics.frontend]: FrontendBackground,
+  [Republics['machine-learning']]: MlBackground,
```
Comment (adrianczekaj) [inline components/Republic/Background.tsx:15]: it's very nice too
I don't know how to go outside thinking only about 'switch'

File: components/Republic/index.tsx [hunk 4]
```diff
@@ -33,21 +34,34 @@ const Block = tw.div`
...
+    case Republics.blockchain:
       return <BlockchainBadge />
-    case 'frontend':
+    case Republics.frontend:
       return <FrontendBadge />
-    case 'machine_learning':
+    case Republics['machine-learning']:
       return <MLBadge />
   }
 }
```
Comment (adrianczekaj) [inline components/Republic/index.tsx:48]: and now I know probably we can change it for one without 'switch'

File: components/Republic/Background.tsx (hunk 3 above)
Comment (hindo) [reply components/Republic/Background.tsx:15]: Time and experience

**Pull request merged**